healthcareai/deploy_supervised_model.py

```diff
@@ -3,6 +3,8 @@
 from healthcareai.common.transformers import DataFrameImputer
 from healthcareai.common import model_eval
 from healthcareai.common import filters
+from healthcareai.common.healthcareai_error import HealthcareAIError
```

Comment:
I think it may be more typical to have something like this in each of your modules:
```
class Error(Exception):
  """Raised when there's a problem with the foo.bar module."""
  pass
```
Then when you anticipate a problem doing something with that module in other code, you reference it like this:
```
from foo import bar
...
try:
  foo = bar.Foo
except bar.Error:
  ...
```